src/back/util/config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Configuration helper related methods.
+ */
+import config from 'config';
+
+/**
+ * Returns config to be used in the front-end
+ */
+export const frontEndConfig = () => {
+  const baseUrl = config.get('baseUrl');
+
+  return {
+    baseUrl,
+    graphqlEndpoint: `${baseUrl}graphql`,
+  };
+};
+
+/**
+ * Checks if current environment is dev.
+ */
+export const isDev = () => {
+  return process.env.NODE_ENV === 'dev';
```

Comment:
You define earlier within another file that **Environment will always be Development unless Production is explicitely defined.** Based on that statement, shouldn't this evaluate to 
```js
return process.env.NODE_ENV !== 'production' 
```